database/seeders/IdentitySeeder.php

```diff
@@ -4,221 +4,235 @@
-                'clusters' => ['disability-and-deaf'],
-            ],
-            [
-                'name' => __('Temporary disabilities'),
-                'description' => __('Such as broken limbs, gestational diabetes'),
-                'clusters' => ['disability-and-deaf', 'reachable-when-mixed'],
-            ],
-            [
-                'name' => __('White'),
-                'clusters' => ['ethnoracial', 'reachable-when-mixed'],
-            ],
-            [
-                'name' => __('Black'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('East Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('South Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Southeast Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Middle Eastern'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Latin American'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('African'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Women'),
-                'clusters' => ['gender', 'gender-and-sexuality'],
-            ],
-            [
-                'name' => __('Men'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'reachable-when-mixed'],
-            ],
-            [
-                'name' => __('Non-binary people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender non-conforming people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender fluid people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('First Nations'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Inuit'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Métis'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Supporters'),
-                'clusters' => ['lived-experience', 'reachable-when-mixed'],
-            ],
-        ];
+        // check if there is a JSON file that has stored data for the seeder
+        if (Storage::disk('seeds')->exists('identities.json')) {
```

Comment:
Added this check for all the seeders sections that I added to prevent them from running in testing or production.

```php
in_array(config('app.env'), ['testing', 'production']) !== true && 
```